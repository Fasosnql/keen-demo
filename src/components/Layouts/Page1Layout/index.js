import React from 'react';

const Page1Layout = () => (
  <div>
    page1
  </div>
);

export default Page1Layout;