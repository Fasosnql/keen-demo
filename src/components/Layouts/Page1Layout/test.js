import React from 'react';
import Page1Layout from './index';

import snapshotTest from 'utils/snapshot-test.util';

snapshotTest('Page1 Layout renders correctly', (
  <Page1Layout/>
));