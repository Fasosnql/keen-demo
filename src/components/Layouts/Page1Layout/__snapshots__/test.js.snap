// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page1 Layout renders correctly 1`] = `
<div>
  page1
</div>
`;
